config.sample.json

```diff
@@ -15,7 +15,7 @@
         "feature_pinning": "labs"
     },
     "default_federate": true,
-    "default_theme": "light",
+    "default_theme": "dharma",
```

Comment:
should we rename dharma to light at this point? Or call dark 'dharma dark' or something?

Reply:
Hmm... Okay, I'll rename `dharma` to `light`.